contracts/pair_stable_bluna/src/contract.rs

```diff
@@ -1542,6 +1542,7 @@ pub fn migrate(deps: DepsMut, env: Env, msg: MigrateMsg) -> Result<Response, Con
                         &config.factory_addr,
                     )?);
             }
+            "1.0.1" => {}
```

Comment:
We do not need this empty action

Reply:
We need to migrate from 1.0.1 without audit fixes to 1.0.1-fix1 with audit fixes. ContractError :: MigrationError will be returned, if I remove `"1.0.1" => {}`.